joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/MemberAccessTests.scala

```diff
@@ -153,4 +153,125 @@ class MemberAccessTests extends CSharpCode2CpgFixture {
       }
     }
   }
+
+  "conditional method access expressions for chained calls" should {
+    val cpg = code("""
+        |namespace Foo {
+        | public class Baz {
+        |   public int Qux() {}
+        |   public Baz Fred(int a) {}
+        | }
+        | public class Bar {
+        |   public static void Main() {
+        |     var baz = new Baz();
+        |     var b = baz?.Fred(1)?.Fred(2)?.Qux();
+        |   }
+        | }
+        |}
+        |""".stripMargin)
+
+    "have correct types and attributes both on the LHS and RHS" in {
+      inside(cpg.assignment.l.sortBy(_.lineNumber).drop(1).l) {
+        case a :: Nil =>
+          inside(a.argument.l) {
+            case (lhs: Identifier) :: (rhs: Call) :: Nil =>
```

Comment:
None of these tests validate the calls chained on the RHS. Please add these, then I'm happy to merge